src/AgGrid.elm

```diff
@@ -1290,6 +1339,22 @@ generateGridConfigAttributes gridConfig =
         ++ configAttributes
 
 
+encodeExcelExportParams : ExcelExportParams -> Json.Encode.Value
+encodeExcelExportParams params =
+    Json.Encode.object
+        [ ( "fileName", Json.Encode.string params.fileName )
+        , ( "columnKeys", Json.Encode.list Json.Encode.string params.columnKeys )
+        ]
+
+
+encodeCsvExportParams : CsvExportParams -> Json.Encode.Value
+encodeCsvExportParams params =
+    Json.Encode.object
+        [ ( "fileName", Json.Encode.string params.fileName )
+        , ( "columnKeys", Json.Encode.list Json.Encode.string params.columnKeys )
+        ]
```

Comment:
Maybe it makes sense to unify the `ExportParams`, what do you think?

Reply:
The possible configuration of the two exports in AgGrid is different. There are some similarities, but they may differ at some point. That's why I've already created them as separate types, making it easier to update the types independently in the future. I'm not sure if this is already required as it creates what appears to be redundant code - but not really much. And theoretically they are different configurations. So I'm not sure if we really should change it

Reply:
Ok, thats a good reason :+1: